abs.yaml

```diff
@@ -1,7 +0,0 @@
-build_env_vars:
-  ANACONDA_ROCKET_ENABLE_PY313 : yes
```

Comment:
Can you enable py314?
`ANACONDA_ROCKET_ENABLE_PY314 : yes`

It's foundational package for py314 buildout.